CONTRIBUTING.md

```diff
@@ -22,22 +26,80 @@ playground rules and follow them during all your contributions.
 
 
 # Setting up Local Environment
-  - Follow instructions on the [Coding in PANOPTES](https://github.com/panoptes/POCS/wiki/Coding-in-PANOPTES).
+  - Follow instructions on the 
+  [Coding in PANOPTES](https://github.com/panoptes/POCS/wiki/Coding-in-PANOPTES).
 
 
 # Testing
  - All changes should have corresponding tests and existing tests should pass after your changes.
- - For more on testing see the [Coding in PANOPTES](https://github.com/panoptes/POCS/wiki/Coding-in-PANOPTES) page.
+ - For more on testing see the 
+ [Coding in PANOPTES](https://github.com/panoptes/POCS/wiki/Coding-in-PANOPTES) page.
 
 
 # Code Formatting
 
 - All Python should use [PEP 8 Standards](https://www.python.org/dev/peps/pep-0008/)
-   - Use a tool such as [yapf](https://github.com/google/yapf) to format your
-     files; we'd rather spend time developing PANOPTES and not arguing about
-     style.
+   - Line length is set at 120 characters instead of 80
```

Comment:
100, not 120, right?

Reply:
Arg...this and the comment below are the same as what you saw happening with the serial testing PR where it is pulling in the old items.